en/docs/edit-the-code/diagnostics.md

```diff
@@ -1 +1,15 @@
 # Diagnostics
+
+The diagnostics show you the syntax and semantic errors in the source code. Varieties of diagnostics such as warnings and errors will be shown. They will show up in multiple places:
+
+* In the Status Bar, there is a summary of all errors and warnings counts.
+* You can click on the summary or view the PROBLEMS panel with a list of all current errors.
+* If you open a file that has errors or warnings, they will be rendered inline with the text and in the overview ruler.
```

Comment:
```suggestion
* Errors or warnings in a Ballerina file will be rendered inline with the text and in the overview ruler.
```

Reply:
Curious how errors and warning are rendered inline. Are we referring to the light bulb?
Also, will see if we can avoid the term `Overview ruler`.

Reply:
Redered inline here means the `red/yellow color underline` you will see if there's an error in your code.
![Screenshot from 2023-01-09 14-22-48](https://user-images.githubusercontent.com/11285165/211270498-295cc392-732a-4118-8799-6e1c9c1622e9.png)

Reply:
ack